README.md

```diff
@@ -17,7 +17,7 @@ Differ builds on the amazing work done by [JustAssembly, licensed under Apache 2
 Distributed as a .NET tool so install using the following
 
 ```
-dotnet tool install assembly-differ
+dotnet tool install -g assembly-differ
```

Comment:
left this one out by choice, [`dotnet local tools`](https://docs.microsoft.com/en-us/dotnet/core/tools/local-tools-how-to-use) should be the default IMO :)

Reply:
Mind reverting this one?